Doc/library/functions.rst

```diff
@@ -1545,6 +1556,9 @@ are always available.  They are listed here in alphabetical order.
    Rather than being a function, :class:`tuple` is actually an immutable
    sequence type, as documented in :ref:`typesseq-tuple` and :ref:`typesseq`.
 
+   .. versionchanged:: 3.7
+      *iterable* is now a positional-only parameter.
```

Comment:
As with the note on list, and for the same reasons, this change isn't needed, and while the documented name matches ("iterable" in the signature and the versionchanged note), it's misleading/confusing; the old keyword name was "sequence", so "iterable" was never a keyword parameter to begin with.